db/migrate/20181114141354_add_event_data_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddEventDataToEvents < ActiveRecord::Migration[5.2]
+  def change
+    add_column :events, :event_data, :jsonb
```

Comment:
the event prefix is implied by the table name - a nit pick but i'll help when i'm typing `event.event_data` vs `event.data` long term.